### DISCUSSION TITLE
How to get value in @map of enum value?

### DISCUSSION THREAD

**Discussion by qwang07** [prisma/prisma#19604] [Q&A]
Prisma schema only supports English. If I want to set non-English enumeration values, I have to use the `@map` method. However, Non-English enumeration values can be correctly stored in the database, it cannot be queried.

e.g. (schema.prisma)
```prisma
enum ApplicationType {
    DRAFT      @map("草稿")
    REVIEWING  @map("审核中")
    APPROVED   @map("已通过")
    DECLINED   @map("已拒绝")
}
```
When I execute the query, only English values are returned. How can I get value in `@map` method?

**Comment by ludralph** [comment]
Hi @qwang07 👋 

At this moment, this is the expected behavior however we do have a feature request to get mapped enum values as can be seen below. Do add your use case to the issue and leave a 👍  to help us prioritize it.

- https://github.com/prisma/prisma/issues/8446

**Reply by ludralph** [reply]
Hey @qwang07,

Do you have any more questions about this issue? If no, it would be really helpful if you could quickly **mark this Discussion as _answered_** to indicate that your issue has been resolved. 

In case you do have any outstanding questions, feel free to let us know and we'd be happy to help!

**Reply by ludralph** [reply]
We are closing this Discussion due to inactivity. Please open a new Discussion if you have a similar problem 🙏
